Assets/Scripts/ConnectionManagement/Unity.BossRoom.ConnectionManagement.asmdef

```diff
@@ -10,7 +10,8 @@
         "GUID:5540e30183c82e84b954c033c388e06c",
         "GUID:59b3bd604d0445f583783872c3e648fc",
         "GUID:4c3f49d89436d478ea78315c03159dcc",
-        "GUID:f2d49d9fa7e7eb3418e39723a7d3b92f"
+        "GUID:f2d49d9fa7e7eb3418e39723a7d3b92f",
```

Comment:
Phil changed this for his other assemblies, can we use name based references instead of guids please? guids make it hard to review (and now that we're mostly done with our assembly refactor, we shouldn't need guids anymore)